.changeset/pink-countries-camp.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Hmmm I set it to default update all the packages available in the script, but I wonder if I should manually add some. I know we have perseus-score that seems like it should be in here?